src/content/docs/how-to-setup-freecodecamp-locally.mdx

```diff
@@ -207,7 +207,7 @@ Once you have the prerequisites installed, you need to prepare your development
 
 ## Clone your Fork from GitHub
 
-You should already have a fork of freeCodeCamp on GitHub. If not, [follow these instructions to fork the repository](/how-to-setup-freecodecamp-locally?id=fork-the-repository-on-github). Next, you will need to [clone](https://help.github.com/articles/cloning-a-repository/) your fork to your local machine.
+You should already have a fork of freeCodeCamp on GitHub. If not, [follow these instructions to fork the repository](https://www.freecodecamp.org/news/how-to-fork-a-github-repository/). Next, you will need to [clone](https://help.github.com/articles/cloning-a-repository/) your fork to your local machine.
```

Comment:
Please check the requirements of the issue. You are linking to a news article which is our of context.